websites/M/MatrixBots/presence.ts

```diff
@@ -0,0 +1,90 @@
+interface LangStrings {
+    watching: string,
+  }
+
+const matrixPresence = new Presence({
+    clientId: "748098665033498735"
+  }),
+  getStrings = async (): Promise<LangStrings> => {
+    return matrixPresence.getStrings(
+      {
+        watching: "general.watching"
+      },
+      await matrixPresence.getSetting("lang")
+    );
+  },
+  matrixBrowsing = Math.floor(Date.now() / 1000);
+let matrixSearch: HTMLInputElement, matrixTitle: Element,
+    strings: Promise<LangStrings> = getStrings(),
+    oldLang: string = null;
+
+matrixPresence.on("UpdateData", async() => {
+    const matrixPData: PresenceData = {
+        largeImageKey: "logo",
+    },
+    matrixPage = window.location.pathname,
+    matrixHost = document.location.hostname;
+
+    matrixPData.startTimestamp = matrixBrowsing;
+
+    const newLang = await matrixPresence.getSetting("lang")
+
+    if (!oldLang) {
+        oldLang = newLang;
+      } else if (oldLang !== newLang) {
+        oldLang = newLang;
+        strings = getStrings();
+      }
+
+    if (matrixHost == "www.matrixbots.xyz") {
+        if (matrixPage == "/") {
+            matrixPData.buttons = [
+                {
+                    label: "Visit",
+                    url: "https://www.matrixbots.xyz/"
+                }, {
+                    label: "Discord",
+                    url: "https://discord.gg/BKP2aSKYJR"
+                }
```

Comment:
both buttons not allowed...
![image](https://user-images.githubusercontent.com/34792838/110391576-f415fa80-8067-11eb-87eb-67c2e4913d06.png)